src/vmm/src/logger/metrics.rs

```diff
@@ -1239,7 +1239,17 @@ mod tests {
 
     #[test]
     fn test_init() {
-        let m = &METRICS;
+        // let m = &METRICS;
+        // Now that we have moved logger to vmm, this test
+        // has a conflict with the vmm_config test `test_init_metrics`
+        // which also uses "METRICS" and tests fail with an already
+        // initialized error.
+        // This test is to validate the init() which doesn't require
+        // using METRICS specifically. So, to avoid the conflict we
+        // use a local Metrics to test init() instead of the global
+        // "METRICS"
+        let m: &Metrics<FirecrackerMetrics, FcLineWriter> =
+            &Metrics::<FirecrackerMetrics, FcLineWriter>::new(FirecrackerMetrics::new());
```

Comment:
nit: This seems fairly verbose. Does something like the following work?

```suggestion
        let m: &Metrics<_, FcLineWriter> = &Metrics::new(FirecrackerMetrics::new());
```

Reply:
The simplest (my preferred and fewest characters) would be:
```rust
        let m = &Metrics::<_, FcLineWriter>::new(FirecrackerMetrics::new());
```

Reply:
done